common/lib/xmodule/xmodule/modulestore/tests/test_semantics.py

```diff
@@ -0,0 +1,119 @@
+from xmodule.modulestore.tests.utils import PureModulestoreTestCase, VersioningModulestoreBuilder, MongoModulestoreBuilder, SPLIT_MODULESTORE_SETUP
+from xmodule.modulestore.exceptions import ItemNotFoundError
+from xmodule.modulestore import ModuleStoreEnum
+from xmodule.modulestore.tests.factories import CourseFactory
+
+class AboutItemSemantics(PureModulestoreTestCase):
+    """
+    Verify the behavior of About items (which are intentionally orphaned, DIRECT_ONLY/autopublished)
+    blocks intended to store snippets of course content.
+    """
+
+    __test__ = False
```

Comment:
Assuming this means that this class shouldn't be tested when tests are run (and that `__test__ = True` means the opposite)